From e23985aa9123b5dda2c7fe6d5205356d3fc4f5b7 Mon Sep 17 00:00:00 2001
From: Noriko Hosoi <nhosoi@redhat.com>
Date: Thu, 7 Jul 2016 16:38:13 -0700
Subject: [PATCH 1/3] Ticket #48755 - moving an entry could make the online
 init fail

Description: The upgrade script template 91reindex.pl.in had a syntax error.
See also Bug 1353592 - Setup-ds.pl --update fails

https://fedorahosted.org/389/ticket/48755

Note: one character fix.
(cherry picked from commit aa64641d1974bb52fc4d02808362e76dd86d9cd0)
---
 ldap/admin/src/scripts/91reindex.pl.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/admin/src/scripts/91reindex.pl.in b/ldap/admin/src/scripts/91reindex.pl.in
index c861f64..99b08e3 100644
--- a/ldap/admin/src/scripts/91reindex.pl.in
+++ b/ldap/admin/src/scripts/91reindex.pl.in
@@ -12,7 +12,7 @@ sub runinst {
     # rdn-format value.  See $rdn_format set below.
     # If equal to or greater than this value, no need to reindex.
     # If it needs to be unconditionally reindexed, set 0.
-    my @rdnconditions = (4)
+    my @rdnconditions = (4);
 
     my $config = $conn->search("cn=config", "base", "(objectclass=*)");
     if (!$config) {
-- 
2.4.11

